velox/functions/prestosql/InPredicate.cpp

```diff
@@ -239,6 +242,19 @@ class InPredicate : public exec::VectorFunction {
           return filter_->testInt64(value);
         });
         break;
+      case TypeKind::REAL:
+        applyTyped<float>(rows, input, context, result, [&](double value) {
```

Comment:
Can these `double` be `float` ? Also for instances below

Reply:
Good catch, @ericyuliu we need to accept `float` here, normalize it as `float`, then do a `reinterpret_cast` as `uint32_t` and then extend it into `int64_t`.  Otherwise this code could be buggy.

Reply:
Updated, thanks for the suggestion!

Reply:
seems there is a delay for the pushed change to show up in PR